newIDE/app/src/CompactPropertiesEditor/PropertiesMapToCompactSchema.js

```diff
@@ -0,0 +1,437 @@
+// @flow
```

Comment:
Can you explain if for this file, there are any big changes compared to the existing PropertiesMapToSchema?

Reply:
Perfect then 👍

Reply:
I don't think I can make it generic since the input is the same between the 2 properties editors. So I added a comment to explain